tests/Auth0Test.php

```diff
@@ -84,6 +84,34 @@ public function testThatExchangeReturnsFalseIfNoCodePresent()
         $this->assertFalse( $auth0->exchange() );
     }
 
+    /**
+     * Test that the exchanges succeeds when there is both and access token and an ID token present.
```

Comment:
Is this doc comment copied from elsewhere? I was confused as it doesn't appear to relate to the name of the test.